lib/validators/omnivore.js

```diff
@@ -38,7 +41,52 @@ module.exports = function validateOmnivore(opts, callback) {
       if (size > limits.max_filesize) {
         return callback(invalid('File is larger than ' + prettyBytes(limits.max_filesize) + '. Too big to process.'));
       }
-      callback();
+      //only look at files that passed above file size test
+      var vectorfiles = ['csv','shp', 'kml', 'gpx', 'geojson', 'topojson'];
+      //skip non-vector files
+      if(vectorfiles.indexOf(filetype) < 0){
+        callback();
+      }
+      return validateGdal(files[0], callback);
     });
   });
+
+  function validateGdal(file, callback){
```

Comment:
Looks good @BergWerkGIS -- my only other style advice would be that usually I would not split out a function like this that unless:
- it is going to be called from several places, or
- I am going to write explicit unit tests for the function

In this case I think it is simpler to read if you just put this functionality in-line with the rest of the script.